endpoints/feature-extraction-service.yml

```diff
@@ -0,0 +1,130 @@
+openapi: "3.0.2"
+x-explorer-enabled: false
+x-samples-enabled: false
+info:
+  version: 0.1.0
+  title: Feature Extraction Service
+  description: Deep learning feature extraction for correspondence matching.
+  contact:
+    name: Fantasmo Support
+    email: hello@fantasmo.io
+paths:
+  /features.extract:
```

Comment:
These REST paths with periods look weird to me. Is this following some standard? Why not just `extract/`? Or, if we can do multiple things with features, `features/extract/`?

Reply:
it's an RPC style call, not REST.